rcongui/src/components/Header/header.js

```diff
@@ -219,12 +219,21 @@ const Header = ({ classes }) => {
                             Players
                           </Link>
                         </MenuItem>
-                        <MenuItem onClick={handleClose}><Link color="inherit" component={RouterLink} to="/logs">
+                        <MenuItem onClick={handleClose}>
+                          <Link color="inherit" component={RouterLink} to="/logs">
                             Logs
-                          </Link></MenuItem>
-                          <MenuItem onClick={handleClose}><Link color="inherit" component={RouterLink} to="/combined_history">
+                          </Link>
+                        </MenuItem>
+                        <MenuItem onClick={handleClose}>
+                          <Link color="inherit" component={RouterLink} to="/combined_history">
                             Combined
-                          </Link></MenuItem>
+                          </Link>
+                        </MenuItem>
+                        <MenuItem>
+                          <Link color="inherit" component={RouterLink} to="/player">
```

Comment:
I don't think we need to add a direct link to it, do we? In the end we just want that page to show if you click on the steam id of a guy from the Live view or the History view, that's all.

Initially I was thinking of using this: https://material-ui.com/components/drawers/#drawer

Because if you need to navigate to a new page it's going to be crappy UX.
You find players from the Live page or the History page, so if you navigate out of those page you'll lose your search.

Reply:
This link was previously a link to the pai output this is why I replaced it

Reply:
and i wanted to add a chat

Reply:
You mean, to PM the player in game?

Unfortunately the API of the game doesn't allow it. I keep asking for it, we could build lots of stuff with that,

Reply:
no no  much simpler just to add some comment on the player but i m not convinced yet by my idea